TMSNet07.Homework4.Calculator/TMSNet07.Homework4.Calculator/Program.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TMSNet07.Homework4.Calculator
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            char repiat = 'y';
+
+            while (Char.ToUpper(repiat) == 'Y')  
+            {
+                double firstValue;
+                double secondValue = 0;
+                string oper;
+
+                Console.WriteLine("Введите операцию: +, -, *, /, %, sqr, sqrt, pow");
+                oper = Console.ReadLine();
+                Console.WriteLine("Введите первое значение: ");
+                firstValue = Convert.ToDouble(Console.ReadLine());
+                
+                if(oper != "sqr" & oper != "sqrt")
```

Comment:
```suggestion
                if(oper != "sqr" && oper != "sqrt")
```
If oper is `sqr`, it doesn't make sense to check whether it is `sqrt`